### PR TITLE
[gui] adds new info label Player.SeekStepSize

### PR DESCRIPTION
**Pull request by xhaggi**
This will updates the seek offset in CGUIInfoManager for every call to CSeekHandler::Seek() if using smart seeking which gives us the possibility to output the seek step size before the seek is processed. 

@da-anda mind taking a look .. might we use a different GUI info var as `Player.SeekOffset` e.g. `Player.SeekStepSize`?

### CODE REVIEW

**Comment by xhaggi** [comment]
BTW should we do this for analog as well?

**Comment by da-anda** [comment]
hmmm, a separate var might be nice, or at least a way for skins to differ between smart seeking and other seek methods, because I don't think the step size info is relevant with fixed seek steps. Also we have to think about simmulating the old behavior - when I do so I probably don't need to see a "+30 sec" huge on screen while with smart seeking it's a necessity. So maybe some "seekMode" var?

**Comment by xhaggi** [comment]
@da-anda agree and I've added a new info label `Player.SeekStepSize` which holds the current seek step size if using smart step based skipping. It's currently displayed in Confluence just like the `Player.SeekOffset` label. IMO it's up to our skin devs to change the DialogSeekBar design ;)

File: xbmc/utils/SeekHandler.cpp [hunk 1]
```diff
@@ -141,7 +141,10 @@ void CSeekHandler::Seek(bool forward, float amount, float duration /* = 0 */, bo
       if (g_infoManager.GetTotalPlayTime())
         percentPerSecond = 100.0f / (float)g_infoManager.GetTotalPlayTime();
 
-      m_percent = m_percentPlayTime + percentPerSecond * GetSeekSeconds(forward);
+      int seekSeconds = GetSeekSeconds(forward);
+      g_infoManager.SetSeekStepSize(seekSeconds);
```
Comment (da-anda) [inline xbmc/utils/SeekHandler.cpp]: shouldn't we only define this if more than one seek step is available (at least for now)?

Comment (xhaggi) [inline xbmc/utils/SeekHandler.cpp]: it makes no different IMO as Player.SeekOffset is displayed after seek is performed and Player.SeekStepSize becomes empty.

**Comment by ronie** [comment]
skin changes are ok with me.

**Comment by xhaggi** [comment]
@da-anda any objections apart from your question?

**Comment by da-anda** [comment]
@xhaggi - nope, good to go I think, haven't tested it though

**Comment by xhaggi** [comment]
jenkins build this please

**Pull request merged**